Include/NRIDescs.h

```diff
@@ -1444,10 +1444,15 @@ NRI_ENUM
     D3D11,
     D3D12,
     VK,
+    MTL,
 
     MAX_NUM
 );
 
+// defined in apple framework
+#undef INTEL
+#undef AMD
```

Comment:
looks like these are #defined in apple framework.

Reply:
I don't mind to add these undefs to the interface.